tuf/utils/role_sort.go

```diff
@@ -0,0 +1,28 @@
+package utils
+
+import (
+	"strings"
+)
+
+// RoleList is a list of roles
+type RoleList []string
+
+// Len returns the length of the list
+func (r RoleList) Len() int {
+	return len(r)
+}
+
+// Less returns true if the item at i should be sorted
```

Comment:
Go's sort itself is an unstable ordering because even with lexical sorting, technically 2 identical items won't always be sorted the same way around.